lib/routes/receive_payment/receive_payment_page.dart

```diff
@@ -133,6 +133,10 @@ class _ReceivePaymentPageState extends State<ReceivePaymentPage> {
     switch (method) {
       case PaymentMethod.bitcoinAddress:
         return ReceiveBitcoinAddressPaymentPage.pageIndex;
+      case PaymentMethod.bolt12Offer:
+        // TODO: Add a BOLT12 offer to the Lightning address page
```

Comment:
Linter warning: `To-do comment doesn't follow the Flutter style.`
```suggestion
        // TODO(dangeross): Add a BOLT12 offer to the Lightning address page
```